### DISCUSSION TITLE
rotate ebsd data to X out of plane

### DISCUSSION THREAD

**Discussion by amirtopaz** [mtex-toolbox/mtex#556] [Ask Anything]
Hi, 
I'm trying to rotate the ebsd data so that x will be out of plane, but it doesn't work.
I've read somewhere that it is impossible to rotate the ebsd data that way, and x has to be directed to east or west. is it true? if isn't, how can I rotate the data?

Thanks,
Amir

**Comment by ralfHielscher** [comment]
Hi Amir,

if you have rotated your EBSD data such that x points out of the plane how
are you gonna plot your data?

Ralf.

**Reply by amirtopaz** [reply]
this is what I want to find out...
the data I'm investigating is on the YZ plane (perpendicular to lineation), so that X is out of plane, and YZ are on the east-west, north-south directions. But since in MTEX the X directions is east and Y is out of plane, plots of pole figures, IPFs and m2m are misoriented relative to the reference frame...

**Comment by kilir** [comment]
Your X,Y,Z are a arbitrary names which are not the the same as x,y,z in mtex. However, of course you can rotate your pole figues or call your reference direntions for ipfs by differen names. 
Hope that helps.
Rüdiger
